rootfs/runner/init

```diff
@@ -47,9 +48,8 @@ case "$1" in
 			command="$(ruby -e "require 'yaml';puts (YAML.load_file('.release')['default_process_types'] || {})['$2']")"
 		fi
 		;;
-
 	*)
-		command="$@"
+		command=$*
```

Comment:
what was the shellcheck error here? Just curious on why it would recommend `$*` instead of `$@`.